addon/content/messageHeaderOptions.jsx

```diff
@@ -130,7 +130,7 @@ class _MessageHeaderOptions extends React.PureComponent {
       document.addEventListener("keypress", this.keyListener);
     }
 
-    this.setState({expanded: !this.state.expanded});
+    this.setState({expanded: true});
```

Comment:
This is wrong, since we'll now never get collapsed. This should use the callback functionality suggested in the details of the alert.

Reply:
Thanks, I rewrote it

Reply:
Oh, I figured I should rewrite the commit message, let me do it real quick

Reply:
Done